Sources/TSCUtility/Triple.swift

```diff
@@ -102,6 +103,9 @@ public struct Triple: Encodable, Equatable {
         if string.hasPrefix(ABI.android.rawValue) {
             return ABI.android
         }
+        if string == ABI.msvc.rawValue {
```

Comment:
This should be `string.hasPrefix(ABI.msvc.rawValue)`.  This is a valid triple: `x86_64-unknown-windows-msvc19.26.28806-macho`